runtime/error.c

```diff
@@ -235,6 +240,15 @@ static void printBacktraceLine(const char *filename, int64_t lineNumber, const c
     fflush(fp);
 }
 
+static bool isPrefix(const char *prefix, const char *str) {
+    while (*++prefix == *++str) {
```

Comment:
Don't think it's safe to ignore the first character.

```
for (; *prefix == *str && *prefix != '\0'; prefix++, str++) { }
```

Reply:
fixed